daphne_worker/src/durable/reports_processed.rs

```diff
@@ -54,13 +56,24 @@ impl DurableObject for ReportsProcessed {
             state,
             env,
             touched: false,
+            alarmed: false,
         }
     }
 
     async fn fetch(&mut self, mut req: Request) -> Result<Response> {
         let id_hex = self.state.id().to_string();
         ensure_garbage_collected!(req, self, id_hex.clone(), BINDING_DAP_REPORTS_PROCESSED);
 
+        let global_config: DapGlobalConfig = serde_json::from_str(
+            self.env.var("DAP_GLOBAL_CONFIG")?.to_string().as_ref(),
+        )
+        .map_err(|e| Error::RustError(format!("Failed to parse DAP_GLOBAL_CONFIG: {}", e)))?;
+        ensure_alarmed!(
+            self,
+            id_hex.clone(),
+            std::time::Duration::from_secs(global_config.report_storage_epoch_duration)
```

Comment:
nit: `use std::time::Duration`.
```suggestion
            Duration::from_secs(global_config.report_storage_epoch_duration)
```